src/components/TodoComponents/Instance.js

```diff
@@ -1,9 +1,21 @@
 import React from 'react';
-
+import './Todo.css';
 
 const Instance = props => {
+
+    let classNames="item";
+    if(props.item.completed){
+        classNames += "completed";
+    }
```

Comment:
In line 8, don't forget to add a whitespace character at the start of your string to add (`" completed"`).

When applying multiple classes, there must be a space character between two separate class names; otherwise, the two class names will be treated as one combined string and then neither of them will work as a result.